source/game.ai.base.bmx

```diff
@@ -15,8 +15,13 @@ Type TAiBase
 	Field scriptSaveState:string
 	'time in milliseconds for the next "onTick"-call
 	Field nextTickTime:Long
-	'game game minute of the last "onTick"-call
-	Field lastTickMinute:int
+	'last processed game minute (since start) (eg via onTick-call)
+	Field processedGameMinute:Long = -1
```

Comment:
Würde es Sinn ergeben, diese Felder nicht mit im Spielstand zu speichern?

Reply:
Die sind nicht notwendig, sollten aber das Kraut nicht fett machen. Bin da flexibel.

Reply:
Was aber sein kann: sie müssen beim Neustart eines Spiels zurückgesetzt werden (laufendes Spiel, zurück zum Hauptmenü, neues Spiel starten).

Reply:
da wird doch eh neue "TAI" angelegt ?

Reply:
OK. Es war nur in der Vergangenheit immer mal so, dass bei neuem (Folge)Spiel nicht alle Werte zurückgesetzt werden - weil eben nicht immer einfach komplett neue Instanzen erzeugt werden, sondern die globals weiterverwendet werden.